client/driver/env/env.go

```diff
@@ -30,8 +30,8 @@ const (
 
 	// Prefix for passing both dynamic and static port allocations to
 	// tasks.
-	// E.g. $NOMAD_PORT_1 or $NOMAD_PORT_http
-	PortPrefix = "NOMAD_PORT_"
+	// E.g. $NOMAD_IP_1=127.0.0.1:1 or $NOMAD_IP_http=127.0.0.1:80
+	IPPortPrefix = "NOMAD_IP_"
```

Comment:
@iverberk Can we name this NOMAD_ADDR? I think addr sounds better since the value would be a ip:port and not just the IP.